velox/expression/tests/CastExprTest.cpp

```diff
@@ -759,3 +728,14 @@ TEST_F(CastExprTest, decimalToDecimal) {
           DECIMAL(12, 0)),
       true);
 }
+
+TEST_F(CastExprTest, castInTry) {
+  auto data = makeVectorWithNullArrays<StringView>(
```

Comment:
Having both 'data' and 'input' makes this code a bit hard to read. Consider inlining 'data' variable.